spec/models/food_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
```

Comment:
- [ ] The tests are failed, please fix them and make sure that all your tests passed as required for this project :+1: 
![test](https://user-images.githubusercontent.com/6615394/225975699-d5f50f7a-036f-485f-8f19-0dda3c38f70d.png)

Reply:
Hey, @codecaiine , if you read the pull request, I mentioned that tests are based on the seeds data. I'm pretty sure you didn't follow the instruction in Pull Request. Please follow them, otherwise no test will be valid

Reply:
For the next code reviewer, Please run `rails db:seed RAILS_ENV=test` before testing the database. Otherwise, all the tests will fail, because the tests are based on the seeds data. We don't create Data directly from the spec files because it's a bad practice. I know there are methods such as destroy to destroy the data when finishing the test but it doesn't always work. and it messes up database. Thank you